README

```diff
@@ -1,2 +1,3 @@
-We need to do:
-	pyuic4 -o pulnix6740_ui.py pulnix6740.ui
+After cloning this repository, just run "make"!
```

Comment:
Maybe note that this requires access to the WEKA NFS (/cds)

Reply:
Agreed.